source/class/qx/tool/cli/commands/Es6ify.js

```diff
@@ -55,13 +56,23 @@ qx.Class.define("qx.tool.cli.commands.Es6ify", {
   members: {
     async process() {
       await this.base(arguments);
+      const ignoreFileName = ".prettierignore";
+      const ig = ignore();
+      if (fs.existsSync(ignoreFileName)) {
+        ig.add(fs.readFileSync(ignoreFileName).toString());
```

Comment:
Maybe we don't care in this app, but instead of the Sync variants, this could await the async variants of `exists` and `readFile`